testdata/p4_16_samples_outputs/list-compare-first.p4

```diff
@@ -11,6 +11,8 @@ const bool w = true;
 const S s = { 32w10, 32w20 };
 const bool v = true;
 const bool o = false;
+const bool vnot = true;
```

Comment:
Something is definitely awry here, as both `s == x` and `s != x` got constfolded to true...

Reply:
That is weird, because the first time I ran the test it actually was correct.
I will have to look into this.